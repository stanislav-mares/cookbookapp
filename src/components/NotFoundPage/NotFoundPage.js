import React from 'react';
import PropTypes from 'prop-types';

const NotFoundPage = (props) => {
  return(
    <div>
      <h1>Page not found!</h1>
    </div>
  );
}

NotFoundPage.propTypes = {

};

export default NotFoundPage;
