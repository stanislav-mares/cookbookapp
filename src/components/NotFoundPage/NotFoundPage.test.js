import React from 'react';
import enzymeConfig from 'testConfig/enzymeConfig';
import { shallow } from 'enzyme';

import NotFoundPage from 'components/NotFoundPage/NotFoundPage';

describe('<NotFoundPage />', () => {
  it('Testing structure of the NoutFoundPage component', () => {
    const component = shallow(<NotFoundPage />);

    expect(component).toMatchSnapshot();
  });
});
