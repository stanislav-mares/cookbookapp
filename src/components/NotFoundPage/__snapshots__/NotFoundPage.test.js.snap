// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> Testing structure of the NoutFoundPage component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFoundPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Page not found!
      </h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Page not found!",
      },
      "ref": null,
      "rendered": "Page not found!",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Page not found!
        </h1>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Page not found!",
        },
        "ref": null,
        "rendered": "Page not found!",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
